pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -937,7 +937,7 @@ func keyfileWatcher(ctx context.Context, watcher *fswatcher.Watcher, keyfilePath
 			// nodeUpdate(), which is responsible for updating the
 			// IPSec policies and states for all the different EPs
 			// with ipsec.UpsertIPsecEndpoint()
-			nodeHandler.NodeValidateImplementation(*nodediscovery.LocalNode())
+			nodeHandler.AllNodeValidateImplementation()
```

Comment:
Small nit: in the comment above s/NodeValidateImplementation/AllNodeValidateImplementation/